examples/remote_procedure/toxicity_detection/webapp/cypress/e2e/remote_procedure_toxicity_detection.cy.js

```diff
@@ -0,0 +1,54 @@
+describe("Loads remote_procedure_toxicity_detection", () => {
+  it("Makes request for agent", () => {
+    cy.intercept({ pathname: "/request_agent" }).as("agentRequest");
+    cy.visit("/");
+    cy.wait("@agentRequest").then((interception) => {
+      expect(interception.response.statusCode).to.eq(200);
+    });
+  });
+
+  it("Loads correct react elements", () => {
+    cy.visit("/");
+    cy.get('[data-cy="directions-header"]');
+    cy.get('[data-cy="directions-paragraph"]');
+    cy.get('[data-cy="detection-text-area"]');
+    cy.get('[data-cy="submit-button"]');
+  });
+
+  it("Typing and submitting a toxic statement", () => {
+    cy.visit("/");
+    cy.get('[data-cy="detection-text-area"]').as("textArea");
+    cy.get('[data-cy="submit-button"]').as("submitButton");
+
+    cy.get("@textArea").type("I hate bob!");
+    cy.get("@submitButton").click();
+    cy.get('[data-cy="loading-spinner"]');
+
+    cy.get('[data-cy="toxicity-alert"]', { timeout: 25000 }).as(
+      "toxicityAlert"
+    );
+    cy.get("@toxicityAlert").contains(
+      'The statement, "I hate bob!," has a toxicity of:'
+    );
+  });
```

Comment:
A long timeout is required for this as the detoxify model takes a long time to run on the GitHub action (they have slow computers).